biogeochem/EDMortalityFunctionsMod.F90

```diff
@@ -275,18 +276,54 @@ subroutine Mortality_Derivative( currentSite, currentCohort, bc_in, frac_site_pr
             * currentCohort%n
     else
 
-      
-    
        ! Mortality from logging in the canopy is ONLY disturbance generating, don't
        ! update number densities via non-disturbance inducing death
-       currentCohort%dndt= -(1.0_r8-fates_mortality_disturbance_fraction) &
-            * (cmort+hmort+bmort+frmort+smort+asmort) * &
-            currentCohort%n
+
+       ! for plants whose death is not considered disturbance (i.e. grasses),
+       ! need to include all of their mortality here rather than part of it here
+       ! and part in disturbance routine.
+
+       if ( .not. ExemptTreefallDist(currentCohort)) then
+          currentCohort%dndt= -(1.0_r8-fates_mortality_disturbance_fraction) &
+               * (cmort+hmort+bmort+frmort+smort+asmort) * &
+               currentCohort%n
+       else
+          currentCohort%dndt= - (cmort+hmort+bmort+frmort+smort+asmort) * &
+               currentCohort%n
+       endif
```

Comment:
Noticed a super minor white space "issue" and thought this might be good to reduce duplication:
```suggestion
       currentCohort%dndt= -(cmort+hmort+bmort+frmort+smort+asmort) * currentCohort%n
       if ( .not. ExemptTreefallDist(currentCohort)) then
          currentCohort%dndt = (1.0_r8-fates_mortality_disturbance_fraction) * currentCohort%dndt
       endif
```